### ISSUE TITLE
Merge common-identity-types.xsd into common-types.xsd

### ISSUE THREAD

**Issue by phax** [de4a-wp5/xml-schemas#41]
Is there a specific reason to keep these two files separate?
As common-types includes common-identity-types I guess it would be easier to just include it

**Comment by phax** [comment]
Alternatively, maybe if we extract an "idk.xsd" to include `ResponseLookupRoutingInformationType`  and `RequestLookupRoutingInformationType` and that one could just reference `common-types.xsd`

But the "common-identity-types" as only "external" interface does not look consistent

**Comment by aosunacab** [comment]
I might be agree with merge common-identity-types.xsd with common-types.xsd. But the IDK types are included in the common estructe since there won't be used separately by any external component (apart to the Connector)

**Comment by aosunacab** [comment]
You mean in case of an external component interacting directly with de IDK?, because those types are just to get the responses from the IDK using the Connector as proxy.

**Comment by phax** [comment]
Fair point. In that case - if you can merge common-identity-types.xsd in common-types.xsd let's close this issue afterwards.
